project-communications/newsletter/newsletter-20211123.md

```diff
@@ -0,0 +1,103 @@
+
+We hope you are enjoying the beautiful colours of the season, and have lots of nice things to look forward to in the Autumn and Winter. Change is the one thing that seems to stay constant, and it affects everyone differently. So, if anyone is struggling with these challenging times, we hope that the citizen science community can offer you support and connection.
+
+Email [Georgia](mailto:gaitkenhead@turing.ac.uk) to let us know if we can help you with involvement, and do feel free to share this newsletter to anyone who you think would be interested in joining us.
+
+They can sign up by following this link: https://tinyletter.com/AutisticaTuringCitizenScience.
+
+Best wishes for everyone!
+
+## Project News
+The most recent updates on the project and its progress can be found [here](https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/project-management/project-updates).
+
+Meet-ups are being held every 2nd and 4th Thursday of the month, as well as the Platform Development co-working sessions on every 1st and 3rd Tuesday of the month. Everyone is welcome to join! The best way to continue contributing is either through these events or by joining our Slack channel via Open Humans and heading to the #autistica-turing channel. 
+
+### Design work for the AutSPACEs platform
+
+Sophia (a.k.a. 'Agenda Bot') and Soledad have been busy working on user experience research and design system for the AutSPACEs platform. This will help make sure the platform is built on community members' true needs. Sophia and Soledad presented on AutSPACEs [personas](https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/platform-designs/design-research/Personas) and [information architecture](https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/platform-designs/design-architecture) for the most recent meet-up session. You can find information about the design and user research work [here](https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/platform-designs). Anyone is invited to get involved via. our Issues and Pull Requests on GitHub. Take a look at the [Design Milestone](https://github.com/alan-turing-institute/AutSPACEs/issues?q=is%3Aopen+is%3Aissue+milestone%3A%226.+Design+for+MVP%22) to find out more.
+
+### Code of Conduct Presentation
+Susanna and James have been busy working on code of conducts. Susanna presented the first draft of code of conducts for the most recent meet-up session. :tada: You can find the draft [here](https://docs.google.com/document/d/1qhBlUxMVv8F_QGuY1re2xbLLZ-mNmak4TISmC-nFW58/edit). Anyone is invited to get involved via. our Issues and Pull Requests on GitHub. Take a look at the [Moderation Milestone](https://github.com/alan-turing-institute/AutisticaCitizenScience/milestone/27) to find out more.
+
+### Method Paper Presentation to Whitakerlab
+> Needs to be updated
```

Comment:
Georgia delivered a paper to The Whitaker Lab at The Turing Institute about a method combining participatory research, citizen science, and open source. You can view and download the slides for the presentation on [Zenodo](https://zenodo.org/record/5615381).